homeassistant/components/rainmachine/switch.py

```diff
@@ -115,6 +132,23 @@
 SWITCH_TYPE_PROGRAM = "program"
 SWITCH_TYPE_ZONE = "zone"
 
+CV_VALID_PERCENTAGE = vol.Any(vol.All(vol.Coerce(int), vol.Range(min=0, max=100)), None)
+CV_VALID_TEMP_RANGE = vol.Any(
```

Comment:
From a naming standpoint, I wonder if these constants should be named such that they are explicitly denoted as belonging to this new service (otherwise, a browser of the code might see `CV_VALID_TEMP_RANGE` and wonder, "Valid temperature range for what?"

Thoughts?

Reply:
@MartinHjelmare side question: would it be worth adding a `cv.percentage` to `config_validation` and updating elsewhere in the codebase (separate PR of course)? I noticed several integrations that would use it. Would that need an arch discussion?

@bachya That's fine, it could be `WX_DATA_` or ?